src/components/scene/reflection.js

```diff
@@ -155,6 +156,9 @@ module.exports.Component = register('reflection', {
 
   remove: function () {
     this.el.object3D.environment = null;
-    this.removeChild(this.probeLight);
+    if (this.probeLight) {
+      this.el.removeChild(this.probeLight);
+      this.probeLight = null;
```

Comment:
`this.probeLight = null;` is not mandatory, but I find it good code hygiene to remove any reference to a DOM element explicitly. This can help the GC too.

Reply:
This should probably be undefined. null usually represent a missing object. In this case we're resetting the variable. Also for style consistency

Reply:
I removed the line. This doesn't add much. The component is removed anyway, so the DOM element is removed as well.

Reply:
About null vs undefined, please note that in the aframe code base we are currently setting some variable to null in remove like I suggested. In the text component for example:
https://github.com/aframevr/aframe/blob/579473b83772b67c76e1e75513b754defef5c32c/src/components/text.js#L126-L135